src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -863,6 +1032,27 @@ type Coordinates {
     lon: Float
 }
 
+"""
+Plan date time options. Only one of the values should be defined.
+"""
+input PlanDateTimeInput @oneOf {
+    """
+    Earliest departure date time. The returned itineraries should not
+    depart before this instant unless one is using paging to find earlier
+    itineraries. Note, it is not currently possible to define both
```

Comment:
Currently? Is it ever going to be supported?

Reply:
Well in theory we could. We quickly discussed all sorts of weird date time/number of itineraries limitations (partly inspired by the OJP specification).